src/openMVG/rig/Rig.cpp

```diff
@@ -67,17 +67,16 @@ bool Rig::initializeCalibration()
   // check that there are cameras
   assert(_vLocalizationResults.size()>0);
   
-  // check that all the cameras have the same number of localizationResults
-  const std::size_t sequenceLength = _vLocalizationResults[0].size();
+  // make all the cameras have the same number of localizationResults (equal to the shortest clip)
   const std::size_t nCams = _vLocalizationResults.size();
-  
-  for(std::size_t i = 1; i < nCams; ++i)
   {
-    assert(_vLocalizationResults[i].size() == sequenceLength 
-            && "All cameras must have the same number of images");
+    std::size_t shortestSeqLength(-1);
```

Comment:
@zvrba wouldn't be better
```c++
std::size_t shortestSeqLength = std::numeric_limits<std::size_t>::max();
```
I mean, in the end the effect should be the same but we avoid the overflow